madmom/evaluation/key.py

```diff
@@ -175,29 +301,85 @@ class KeyMeanEvaluation(EvaluationMixin):
         ('correct', 'Correct'),
         ('fifth', 'Fifth'),
         ('relative', 'Relative'),
+        ('relative_of_fifth', 'Relative of Fifth'),
         ('parallel', 'Parallel'),
         ('other', 'Other'),
         ('weighted', 'Weighted'),
     ]
 
     def __init__(self, eval_objects, name=None):
         self.name = name or 'mean for {:d} files'.format(len(eval_objects))
-
+        self.relative_of_fifth_present = False
+        if check_key_eval_objects(eval_objects):
+            self._count_evaluations(eval_objects)
+            if self.relative_of_fifth > 0:
```

Comment:
Shouldn't `self.relative_of_fifth_present` depend only on whether the `eval_objects` compute it or not (i.e., after `check_key_eval_objects`, if `eval_objects[0].relative_of_fifth == True`)?
`self.relative_of_fifth` could be 0 because this error type did not occur, and in this case we want to report it. We only don't want to report it if the `eval_objects` did not compute it in the first place.

Reply:
I think I addressed this comment with the latest commit.